models/pull.go

```diff
@@ -624,7 +631,7 @@ func GetPullRequestByIndex(repoID int64, index int64) (*PullRequest, error) {
 	if err != nil {
 		return nil, err
 	} else if !has {
-		return nil, ErrPullRequestNotExist{0, repoID, index, 0, "", ""}
+		return nil, ErrPullRequestNotExist{0, 0, 0, repoID, "", ""}
```

Comment:
The arguments were provided in the wrong order. Sorry, maybe this should have been in a separate PR.